src/typeahead/index.js

```diff
@@ -36,8 +43,7 @@ var Typeahead = React.createClass({
       defaultValue: "",
       placeholder: "",
       onOptionSelected: function(option) {},
-      onKeyDown: function(event) {},
-      filterOption: null
```

Comment:
It's a stylistic thing, mostly. I find specifying `null` or `undefined` as defaults noisy, but I can restore it if you prefer to have explicit defaults listed for all non-required options.